### PR TITLE
Add mask variant

### PR DESCRIPTION
**Pull request by balt-is-you-and-shift**
Add the `:mask` variant.
This variant, instead of rendering the sprite overlaid on the previous tiles, makes the pixels in the sprite's bounds but not in the sprite's alpha transparent. 
Example:
image

### CODE REVIEW

File: src/cogs/render.py [hunk 1]
```diff
@@ -103,9 +103,23 @@ async def render(
...
                         else:
-                            imgs[frame].paste(sprite, (x * constants.DEFAULT_SPRITE_SIZE + padding - x_offset, y * constants.DEFAULT_SPRITE_SIZE + padding - y_offset), mask=sprite)
+                            imgs[frame].paste(
+                                sprite, 
+                                (
+                                    x * constants.DEFAULT_SPRITE_SIZE + padding - x_offset,
+                                    y * constants.DEFAULT_SPRITE_SIZE + padding - y_offset
+                                ), 
+                                mask=sprite
+                            )
```
[the remark this answers is not in the document]
Comment (balt-is-you-and-shift) [reply src/cogs/render.py:125]: alright fine, minus the typo on line 107

[the remark this answers is not in the document]
Comment (balt-is-you-and-shift) [reply src/cogs/render.py:125]: damnit indentation

[the remark this answers is not in the document]
Comment (balt-is-you-and-shift) [reply src/cogs/render.py:125]: fixed indentation

[the remark this answers is not in the document]
Comment (RocketRace) [reply src/cogs/render.py:125]: the indentation was right, you just forgot to delete the else

**Comment by RocketRace** [comment]
I've merged this! Sorry for being late

**Pull request closed without merging**